edx-platform/pearson-vue-theme/lms/student-account/login_and_register.html

```diff
@@ -107,29 +75,8 @@ <h3>
 </div>
 
 
-    <button type="submit" class="action action-primary action-update js-login login-button"><%- gettext("Sign in") %></button>
-
-    <% if ( context.providers.length > 0 && !context.currentProvider) { %>
-    <div class="login-providers">
-        <div class="section-title lines">
-            <h3>
-                <span class="text"><%- gettext("or sign in with") %></span>
-            </h3>
-        </div>
-
-        <% _.each( context.providers, function( provider ) {
-            if ( provider.loginUrl ) { %>
-                <button type="button" class="button button-primary button-<%- provider.id %> login-provider login-<%- provider.id %>" data-provider-url="<%- provider.loginUrl %>">
-                    <div class="icon <% if ( provider.iconClass ) { %>fa <%- provider.iconClass %><% } %>" aria-hidden="true">
-                        <% if ( provider.iconImage ) { %>
-                            <img class="icon-image" src="<%- provider.iconImage %>" alt="<%- provider.name %> icon" />
-                        <% } %>
-                    </div>
-                    <span aria-hidden="true"><%- provider.name %></span>
-                    <span class="sr"><%- _.sprintf( gettext("Sign in with %(providerName)s"), {providerName: provider.name} ) %></span>
-                </button>
-            <% }
-        }); %>
-    </div>
-    <% } %>
-</form>
+<script type="text/javascript">
```

Comment:
Why is this necessary? @luismorenolopera

Reply:
to change the title when clicking on the change password modal @Squirrel18

Reply:
Is it required for this? Is there any other way to do it?

Reply:
The page does not change when it is clicked, because it is a modal of the login page, so it cannot be changed using the block for this @Squirrel18

Reply:
By block, I mean:

`<%block name="pagetitle">${_("Sign in or Register")}</%block>`